prody2/protocols/protocol_atoms.py

```diff
@@ -92,8 +92,8 @@ def selectionStep(self, inputFn):
         ag = prody.parsePDB(inputFn, alt='all')
         selection = ag.select(str(self.selection))
 
-        LOGGER.info("%d atoms selected from %d" % (selection.numAtoms(), 
-                                                          ag.numAtoms()))
+        logging.info("%d atoms selected from %d" % (selection.numAtoms(),
```

Comment:
I thin is is better to:

1.- use a specific logger for the module. This will allow identifying in the log who has generated the log line if configured.

2.- the protocol object already has built-in methods for logging:
   self.info()
   self.warning()
   self.error()
   self.debug()